charts/pxc-db/templates/cluster.yaml

```diff
@@ -1,6 +1,9 @@
-apiVersion: pxc.percona.com/v1
+apiVersion: pxc.percona.com/{{ .Chart.ApiVersion }}
 kind: PerconaXtraDBCluster
 metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+        {"apiVersion":"pxc.percona.com/{{ .Chart.ApiVersion }}","kind":"PerconaXtraDBCluster"}
```

Comment:
normally you wouldn't see a last-applied-configuration and let Kubernetes do that itself.   What's the reason for adding this here ?

Reply:
We are facing the following error without the annotations 

`    - lastTransitionTime: "2020-01-20T18:48:17Z"
      message: 'wrong PXC options: set version: new version: Malformed version: '
      reason: ErrorReconcile
      status: "True"
      type: Error`